src/grt/grt-wave_options.adb

```diff
@@ -0,0 +1,65 @@
+--  GHDL Run Time (GRT) -  command line options.
+--  Copyright (C) 2002 - 2014 Tristan Gingold
+--
+--  GHDL is free software; you can redistribute it and/or modify it under
+--  the terms of the GNU General Public License as published by the Free
+--  Software Foundation; either version 2, or (at your option) any later
+--  version.
+--
+--  GHDL is distributed in the hope that it will be useful, but WITHOUT ANY
+--  WARRANTY; without even the implied warranty of MERCHANTABILITY or
+--  FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+--  for more details.
+--
+--  You should have received a copy of the GNU General Public License
+--  along with GCC; see the file COPYING.  If not, write to the Free
+--  Software Foundation, 59 Temple Place - Suite 330, Boston, MA
+--  02111-1307, USA.
+--
+--  As a special exception, if other files instantiate generics from this
+--  unit, or you link this unit with other files to produce an executable,
+--  this unit does not by itself cause the resulting executable to be
+--  covered by the GNU General Public License. This exception does not
+--  however invalidate any other reasons why the executable file might be
+--  covered by the GNU Public License.
+
+with Grt.Types; use Grt.Types;
+with Grt.Vstrings; use Grt.Vstrings;
+with Grt.Errors; use Grt.Errors;
+
+package body Grt.Wave_Options is
+
+   procedure Print_Context (Line_Context : Line_Context_Acc;
+                            Severity : Severity_Type) is
+      Lineno_Str : String (1 .. Value_String_Size);
+      First : Natural;
+   begin
+      case Severity is
```

Comment:
I am using geany, but the indentation I use for 'when' is actually a 
personnal code style.

Fixed


Le 23. 07. 16 à 12:28, tgingold a écrit :
>
> In src/grt/grt-wave_options.adb 
> <https://github.com/tgingold/ghdl/pull/121#discussion_r71970755>:
>
> > +--  covered by the GNU General Public License. This exception does not
> > +--  however invalidate any other reasons why the executable file might be
> > +--  covered by the GNU Public License.
> > +
> > +with Grt.Types; use Grt.Types;
> > +with Grt.Vstrings; use Grt.Vstrings;
> > +with Grt.Errors; use Grt.Errors;
> > +
> > +package body Grt.Wave_Options is
> > +
> > +   procedure Print_Context (Line_Context : Line_Context_Acc;
> > +                            Severity : Severity_Type) is
> > +      Lineno_Str : String (1 .. Value_String_Size);
> > +      First : Natural;
> > +   begin
> > +      case Severity is
>
> Indentation of 'when' is not correct.
> (Which editor do you use ?)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/tgingold/ghdl/pull/121/files/4bf3293563ba8e95cc35992d029ff26c4f4e67e5#r71970755>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ATNePjqKMC940_D3LDRcrudP-Hec-yNHks5qYeyzgaJpZM4JTTAB>.
>